fl4health/clients/evaluate_client.py

```diff
@@ -42,9 +41,9 @@ def __init__(
         self.data_loader: DataLoader
         self.criterion: _Loss
         self.global_loss_meter = LossMeter.get_meter_by_type(loss_meter_type)
-        self.global_metric_meter = MetricMeter.get_meter_by_type(self.metrics, metric_meter_type, "global_eval_meter")
+        self.global_metric_meter = MetricManager(self.metrics, "global_eval_meter")
```

Comment:
Maybe change the names of these properties to be something like `global_metric_manager` and `local_metric_manager`?